src/main/java/persistence/entity/EntityKey.java

```diff
@@ -0,0 +1,36 @@
+package persistence.entity;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+public class EntityKey implements Serializable {
+
+    private Long longTypeId;
+    private String stringTypeId;
+
+    public EntityKey() {}
+
+    public EntityKey(Long longTypeId, String stringTypeId) {
+        this.longTypeId = longTypeId;
+        this.stringTypeId = stringTypeId;
+    }
+
+    // equals()와 hashCode()를 반드시 구현
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        EntityKey that = (EntityKey) o;
+        return longTypeId.equals(that.longTypeId) && stringTypeId.equals(that.stringTypeId);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(longTypeId, stringTypeId);
+    }
```

Comment:
```suggestion
    @Override
    public int hashCode() {
        return hashCode;
    }
```

검색이 자주된다고 본다면 효율을위해 미리 상태로 가지고 있어도 좋을것 같네요 ㅎ

Reply:
메서드가 불러질때마다 매번 계산되는거는 비효율적인거 같아 보이네요 ㅎㅎ

감사합니다!